src/main/g8/src/main/scala/$package$/api/graphql/$if(add_caliban_endpoint.truthy)$GraphQLApi.scala$endif$

```diff
@@ -70,19 +73,28 @@ object GraphQLApi extends GenericSchema[Any] {
         )
       )
     ) @@
-      maxFields(200) @@               // query analyzer that limit query fields
-      maxDepth(30) @@                 // query analyzer that limit query depth
-      timeout(3 seconds) @@           // wrapper that fails slow queries
-      printSlowQueries(500 millis) @@ // wrapper that logs slow queries
-      apolloTracing                   // wrapper for https://github.com/apollographql/apollo-tracing
+    maxFields(200) @@                                // query analyzer that limit query fields
+    maxDepth(30) @@                                  // query analyzer that limit query depth
+    timeout(3 seconds) @@                            // wrapper that fails slow queries
+    onSlowQueries(500 millis)(slowQueriesHandler) @@ // wrapper that logs slow queries
+    apolloTracing                                    // wrapper for https://github.com/apollographql/apollo-tracing
   }
 
-  val live: ZLayer[ItemRepository with Has[ActorSystem], CalibanError.ValidationError, GraphQLApi] =
-    ZLayer.fromFunctionM { env =>
-      implicit val system = env.get[ActorSystem]
-      implicit val ec     = system.dispatcher
-      val adapter = AkkaHttpAdapter(new PlayJsonBackend)
-      api(env).interpreter.map(interpreter =>
+  val live =
+    ZLayer.fromServicesM[
+      Clock.Service,
```

Comment:
Looks weird to me that we have Clock both as a layer dependency and as environment in the result.
Can it be squashed somehow, so that clock is only a layer dep?